.github/workflows/build-checker.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
       - name: Wait for commit statuses
         id: status
-        uses: WyriHaximus/github-action-wait-for-status@v1
+        uses: WyriHaximus/github-action-wait-for-status@v1.2
```

Comment:
@Crow-EH FYI, `v1` and `v1.2` are a retag of `v1.2.0`

Reply:
You're right, it's not a fix per say. We're just stabilizing in v1.2 :grin:

Reply:
:+1: , just wanted to make sure you'd know the difference :smile: